This one has modification date following the filename.
We are ignoring it, so the canonical form won't have it.
--- filename	2013-09-23 18:41:09.000000000 -0400
+++ filename	2013-09-23 18:41:09.000000000 -0400
@@ -5,7 +5,7 @@ foo
 aaa
 bbb
 ccc
-ddd
+eee
 fff
 ggg
 hhh
